poleno/utils/static/poleno/js/collapsable.js

```diff
@@ -0,0 +1,61 @@
+/* Enables elements to automatically show or hide other elements by bootstrap collapsing.
+ * Works on all elements type, which must contain data-toggle="collapse" and
+ * data-target="`css-selector`" attributes. Collapsable element must contain class="collapse".
+ *
+ * data-target-shown elements are shown, when collapsable elements are hidden. data-target-hidden
+ * elements are shown, when collapsable elements are shown and vice versa.
+ *
+ * By entering url with a hash into browser, content and elements will be automatically collapsed.
+ *
+ * Requires:
+ *  -- JQuery
+ *
+ * Examples:
+ *  <div class="parent">
+ *    <div data-toggle="collapse" data-target="#content">
+ *      <p class="collapse-shown">Collapse</p>
+ *      <p class="is-hidden collapse-hidden">Collapsed</p>
+ *    </div>
+ *    <p id="content" class="collapse" data-target-shown=".collapse-shown"
+ *       data-target-hidden=".collapse-hidden" data-container=".question">
+ *      Content
+ *    </p>
+ *  </div>
+ */
+$(function(){
+	function collapse_show(){
+		var container = $(this).data('container') || 'html';
+		var target_shown = $.map(this.attributes, function(attr){
+			if (attr.name.match("^data-target-shown")) return attr.value;
+		}).join(', ');
```

Comment:
Naco takto zlozito citat tie atributy? V `toggle_changed` to bolo treba pretoze element mohol mat lubovolne vela atributov tvaru `data-hide-target-*`. To mas vsak iba jeden atribut `data-target-shown`. Tak preco nie proste

```var target_shown = $(this).data('target-shown')```

S tym, ze treba overit, co to spravi ak element dany atribut neba zadany.

Reply:
:ok: Ak dany attribut vobec neexistuje `undefined`, alebo ak je to `<empty string>`. tak sa selecte `Object { length: 0, ...}`, takze to netreba nijako osetrovat, jQuery si s tym poradi same.